src/tasks/components/PageBreadcrumbs.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {PureComponent} from 'react';
+
+// Components
+import {
+    FlexBox,
+    FlexDirection,
+  } from '@influxdata/clockface'
+
+type PageInfo = {
+    text: string
+    onClick?: () => void
+}
+
+type Props = {
+    pages: Array<PageInfo>
+}
+
+export default class PageBreadcrumbs extends PureComponent<Props> {
```

Comment:
if we're expecting this component to rerender based on changes made to the parent props, we might consider using a functional component rather than a pure component